lambda_express_bundle/api_gateway.tf

```diff
@@ -1,26 +1,26 @@
+locals {
+  domain_name = var.config["branch"] == "master" ? "${var.subdomain}.bombbomb.io" : "${var.config["branch"]}-${var.subdomain}.dev.bombbomb.io"
+}
```

Comment:
Just pulled this up here instead of down on line 44-ish.

Also removed the sub_subdomain. Why did we ever have that in there? I'll keep an eye out for anyone else using this sadness